### PR TITLE
bootstrap added

### PR DESCRIPTION
**Pull request by mateusuk**

### CODE REVIEW

**Comment by mateusuk** [comment]
background photos and colors were added

File: index.html [hunk 1]
```diff
@@ -4,27 +4,203 @@
...
+  </header>
+
+  <div id="myCarousel" class="carousel slide" data-bs-ride="carousel">
+    <div class="carousel-indicators">
+      <button type="button" data-bs-target="#myCarousel" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
+      <button type="button" data-bs-target="#myCarousel" data-bs-slide-to="1" aria-label="Slide 2"></button>
+      <button type="button" data-bs-target="#myCarousel" data-bs-slide-to="2" aria-label="Slide 3"></button>
+    </div>
+    <div class="carousel-inner">
+      <div class="carousel-item active" style="background-image: url(/images/planets2.jpg); background-size: cover;">
```
Comment (ljenchik) [inline index.html]:
```suggestion
      <div class="carousel-item active" style="background-image: url(./images/planets2.jpg); background-size: cover;">
```

File: index.html [hunk 2]
```diff
@@ -4,27 +4,203 @@
...
+
+        <div class="container">
+          <div class="carousel-caption text-start">
+            <h1>Example headline.</h1>
+            <p>Some representative placeholder content for the first slide of the carousel.</p>
+            <p><a class="btn btn-lg btn-primary" href="#">Sign up today</a></p>
+          </div>
+        </div>
+      </div>
+      <div class="carousel-item" style="background-image: url(/images/newfoto.jpg); background-size: cover;">
```
Comment (ljenchik) [inline index.html]:
```suggestion
      <div class="carousel-item" style="background-image: url(./images/newfoto.jpg); background-size: cover;">
```

File: index.html [hunk 3]
```diff
@@ -4,27 +4,203 @@
...
+        
+        <div class="container">
+          <div class="carousel-caption">
+            <h1>Another example headline.</h1>
+            <p>Some representative placeholder content for the second slide of the carousel.</p>
+            <p><a class="btn btn-lg btn-primary" href="#">Learn more</a></p>
+          </div>
+        </div>
+      </div>
+      <div class="carousel-item" style="background-image: url(/images/pexels-pixabay-2150.jpg); background-size: cover;">
```
Comment (ljenchik) [inline index.html]:
```suggestion
      <div class="carousel-item" style="background-image: url(./images/pexels-pixabay-2150.jpg); background-size: cover;">
```

**Review by ljenchik — COMMENTED** [review]
better to use relative url to the pictures

**Review by ljenchik — APPROVED** [review]

**Pull request merged**